src/projections/tmerc.cpp

```diff
@@ -488,109 +446,33 @@ static PJ *setup_exact(PJ *P) {
     auto *Q = &(static_cast<struct tmerc_data *>(P->opaque)->exact);
 
     assert(P->es > 0);
-
+    static_assert( PROJ_ETMERC_ORDER == int(AuxLat::AUXORDER),
+                   "Inconsistent orders etmerc vs auxorder" );
     /* third flattening */
     const double n = P->n;
-    double np = n;
 
     /* COEF. OF TRIG SERIES GEO <-> GAUSS */
     /* cgb := Gaussian -> Geodetic, KW p190 - 191 (61) - (62) */
     /* cbg := Geodetic -> Gaussian, KW p186 - 187 (51) - (52) */
     /* PROJ_ETMERC_ORDER = 6th degree : Engsager and Poder: ICC2007 */
```

Comment:
Are these lines still relevant here? Seeing as the lines below are removed in favour of a centralised function I'm wondering if keeping the comment here will be confusing for someone down the line.

Reply:
It's important to keep these comments since they correspond to the terms in E&P (2007).  I've inserted a key given the correspondence between these and the names of the auxiliary latitudes.